tests/test_coreg.py

```diff
@@ -218,20 +218,20 @@ def test_deramping(self):
         # Check that the mean periglacial offset is low
         assert np.abs(np.mean(periglacial_offset)) < 1
 
-        # Try a 0 degree deramp (basically bias correction)
+        # Try a 0 degree deramp (basically vertical shift correction)
         deramp0 = coreg.Deramp(degree=0)
```

Comment:
Deramp should maybe also be in `biascorr.py`?

Reply:
It's delicate, as degree 1 will be affine, but higher degree won't be.
Should we split those somehow?
We could have a `Tilt` class in `coreg.py` (i.e. Deramp with degree=1) and a `Deramp` class in `biascorr.py` which, if called with a degree=1, just passes to `Tilt`.
@adehecq @erikmannerfelt  What do you think?

Reply:
I think it would be confusing to have both Tild and Deramp (plus deramp has basically the same meaning as tilt, as it involves a plane surface).
I'm happy to have Deramp in biascorr, if it can be called the exact same way as the other coreg functions, and we clearly state the difference between the two categories somewhere and summarize all possible options.